remote_attestation/rust/src/message.rs

```diff
@@ -236,25 +227,21 @@ impl ServerIdentity {
             + REPLAY_PROTECTION_ARRAY_LENGTH
             + SIGNATURE_LENGTH
             + SIGNING_ALGORITHM_KEY_LENGTH
-            + 2 * VEC_SIZE_PREFIX_LENGTH
+            + VEC_SIZE_PREFIX_LENGTH
```

Comment:
Why did we lose the 2* here?

Reply:
Because one field with varying length (i.e., additional info) is removed.

Reply:
Added clarifying comment.